guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -203,6 +203,7 @@ Allows for the customization of how the given screen should appear/disappear whe
 - `"slide_from_bottom"` - slide in the new screen from bottom to top
 - `"slide_from_right"` - slide in the new screen from right to left (Android only, resolves to default transition on iOS)
 - `"slide_from_left"` - slide in the new screen from left to right (Android only, resolves to default transition on iOS)
+- `"cupetino"` - iOS like slide in animation (Android only, resolves to default transition on iOS)
```

Comment:
I don't know if `cupertino` sounds better as an animation name here (it's not descriptive from the name). Maybe `slide_out_from_side` / `jump_from_side` would be better here?

Reply:
It's inspired by [Flutter](https://docs.flutter.dev/ui/widgets/cupertino). I think `cupertino` is somewhat original and also explains what it actually means.
If it's a firm no - I can change it something like `slide_from_right_ios`.

Reply:
I know the point, but I don't think this name is not that descriptive as it could be 😕 
Actually, how about just `ios` as the name of the transition?

Reply:
renamed